src/common.rs

```diff
@@ -150,6 +150,11 @@ impl Errors {
     }
 }
 
+/// Trait used to bundle up resources/components for easy registration with the `World`.
+pub trait WorldResourceRegistrator {
```

Comment:
I got a better name, today ;) You're already using "bundle" in the documentation, maybe that'd justify calling the trait `Bundle`?